src/lava_callback.py

```diff
@@ -600,7 +600,8 @@ async def checkout(data: ManualCheckout, request: Request,
                 "tree": treename,
                 "branch": branch,
                 "commit": commit,
-                "url": treeurl
+                "url": treeurl,
+                "tip_of_branch": False
```

Comment:
Custom checkout can also have a commit from the tip of the branch. Why is it set `False`?